notebooks/02_foundations__connection_riemannian_metric.ipynb

```diff
@@ -0,0 +1,2266 @@
+{
```

Comment:
<p>Some remarks:</p><p>- In latex, you need to invert the first quotes: <code></code>parallel vector field"</p><p>- a vector field is a <code></code>parallel vector field *along gamma* if</p><p>^^^the "along gamma" is very important here, vector fields can only be parallel along a curve.</p><p> </p><p>I would also mention that \dot gamma is the tangent vector to the curve gamma.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>agree</p>